### ISSUE TITLE
Using Azure Devops git repository 

### ISSUE THREAD

**Issue by eskaarup** [microsoft/azure-pipelines-jira#10]
I have tried to get this up running within our organization, but it is not clear for me if this integration only supports GitHub and not Azure Devops GIT. 

1. The organization is setup in the Azure Pipelines app manager in Jira - no problem
2. I mapped the stages in my release pipeline - no problem
3. Commits with ticket reference to jira - no problem
3. Running a release pipeline does not update the tickets in Jira with release information. 

Do we need to install another 3'party app in Jira to get  GIT connectivity or is this an error.

**Comment by doerig** [comment]
I'd like to get the answer as well for this question... seems kind of strange to release a azure devops integration that doesn't support its own repos

**Comment by koenj** [comment]
This is related to issue #7, describing the same behavior for BitBucket.

**Comment by Thorocaine** [comment]
I am also curious to know about Azure Repos.

**Comment by mirandaposthumus** [comment]
Based on the responses to the [reviews on Jira](https://marketplace.atlassian.com/apps/1220515/azure-pipelines-for-jira?hosting=cloud&tab=reviews) it's not supported, which is very disappointing,

> Currently there is no official way to link commits in Azure DevOps Git Repo with Jira issues. You can however use third party plugins for that.

**Comment by atrauzzi** [comment]
Yeah, this needs to be supported ASAP. I'm not interested in being forced to use three services when I originally came trying to work with two.

**Comment by rogersmj** [comment]
I just spent a bunch of time troubleshooting this issue -- everything appears to be setup fine, no errors, but there's "No work items" listed on the release and no info appearing in JIRA. Just to come on here and find out that they don't support their own repo host? Ridiculous.

**Comment by spr0ut** [comment]
@rogersmj it's pretty clear this integration is not ready for release. Issues all over the place and no support for their own repo.

**Comment by hjsanchez** [comment]
is there any tentative date for this to be even take this into account? would love to have it for azure repos or any way we can contribute?

**Comment by thurfir** [comment]
I don't work for Microsoft, but Azure repo support seems to be in the work according to the App:

> Support for Azure Repos would be added in the next iteration (in roughly 2-3 weeks).

https://marketplace.atlassian.com/apps/1220515/azure-pipelines-for-jira?hosting=cloud&tab=versions

**Comment by basroovers** [comment]
Although that is good news, we're already 5 weeks in. This tool is useless for us unless Azure Repos are supported.

**Comment by Eddie-Hartman** [comment]
6 weeks later... We still really want this feature.

**Comment by Eddie-Hartman** [comment]
Sooo.... It's looking pretty grim here if you don't want to pay for another 3rd party add-on to use this with AzureDevops. The only functionality that I'm seeing it offering is it will list links to Jira work items when the commits for a release include a Jira work item key when you make a release in AzureDevops. The documentation even has two paid third party plugins having at least one installed as prerequisites to get the full functionality out of this. [Link](https://github.com/microsoft/azure-pipelines-jira/blob/master/tutorial.md#prerequisites)

After linking and creating a release, this is the only functionality that I got:
![image](https://user-images.githubusercontent.com/14089997/64210230-b764b400-ce70-11e9-9397-97244ded9fc3.png)

It just added a couple links to work items from the last CI release. If you compare it to older releases, it will compare the commits in-between as well. We are using a YAML pipeline, so it seems to work ok, but isn't really what we we were hoping out of this integration. We were hoping for more of what the paid add-ons provide as a free integration from Microsoft.

It hasn't updated the ticket statuses though. It may have to be a release created after the integration in order for it to do that, but I kind of doubt that. I still think it requires a paid add-on.

**Comment by ahaeber** [comment]
> Sooo.... It's looking pretty grim here if you don't want to pay for another 3rd party add-on to use this with AzureDevops. The only functionality that I'm seeing it offering is it will list links to Jira work items when the commits for a release include a Jira work item key when you make a release in AzureDevops. The documentation even has two paid third party plugins having at least one installed as prerequisites to get the full functionality out of this. [Link](https://github.com/microsoft/azure-pipelines-jira/blob/master/tutorial.md#prerequisites)
> 
> After linking and creating a release, this is the only functionality that I got:
> ![image](https://user-images.githubusercontent.com/14089997/64210230-b764b400-ce70-11e9-9397-97244ded9fc3.png)

This was definitively a big disappointment. I had expected this integration to be capable of at least showing PR information in JIRA. In hindsight I see that the name "Azure Pipelines for JIRA" gives it a bit away that we should not have expected full Azure Repos support in JIRA.

Hopefully Atlassian and Microsoft Azure DevOps will continue with their cooperation now and create official support for Azure Repos in Atlassian JIRA too.

**Comment by apoorvakrishna66** [comment]
The app documentation says it supports Azure Repos as well but I could not get it to work. Any help is appreciated if I needed to change some config or others as such.

**Comment by aivsim** [comment]
@apoorvakrishna66 at another [link](https://developercommunity.visualstudio.com/comments/936815/view.html) one of MS guys informs us that [v1.0.9 - This version adds the support for Azure Repos to the plugin.](https://marketplace.atlassian.com/apps/1220515/azure-pipelines-for-jira?hosting=cloud&tab=versions) is a misunderstanding. It is still needed to install paid, third party plugins in from Jira marketplace...

**Comment by MianMUAmer** [comment]
Hi there, I have the same issue, at work, we are using Azure Repos and Jira, but can't get git integrations until and unless I use the paid version. Did you guys find a workaround or is there any update from Microsoft regarding this?

**Comment by truncksz** [comment]
Any news?

**Comment by austinwilcox** [comment]
Has there been any update on this? I would love to have these integrations.

**Comment by kamatheuska** [comment]
Any news on this?

**Comment by RobertAKARobin** [comment]
Just adding a plus-one -- our organization would like this too.
